lnrpc/rpc.proto

```diff
@@ -140,6 +140,14 @@ service Lightning {
     */
     rpc GetInfo (GetInfoRequest) returns (GetInfoResponse);
 
+    /** lncli: `getrecoveryinfo`
+    GetRecoveryInfo returns information concerning the recovery mode including
+    whether it's in a recovery mode, whether the recovery is finished, and the
+    progress made so far.
+    */
+    rpc GetRecoveryInfo (GetRecoveryInfoRequest)
```

Comment:
Include in `WalletKit` instead?

Reply:
shouldn't this be accessible w/o needing to compile the subserver?

Reply:
I was aware of the `WalletKit` previously😂. To me it looks like more of manipulating the UTXO state than giving general information?

Reply:
> shouldn't this be accessible w/o needing to compile the subserver?

Definitely, I only mentioned it because we've had plans to deprecate the `Lightning` RPC server and adding new RPCs to it doesn't help. I'm fine with its placement now, but we should decide on an approach and stick with it.